### PR TITLE
Adding Starbucks portal and portal scraper

### PR DESCRIPTION
**Pull request by RocketGod-git**
Fixed. Thanks @amouly for the tip.

### CODE REVIEW

**Comment by amouly** [comment]
Why would you do all this work? you can simply install Chrome extension and download the file of that website:

https://chrome.google.com/webstore/detail/singlefile/mpiodijhokgodhhofbcjdecpffjipkle/related

File: portals/Starbucks/portalscraper/chromedriver.exe
Comment (amouly) [inline portals/Starbucks/portalscraper/chromedriver.exe:1]: This only works for Windows, please avoid adding this dependency.

Comment (RocketGod-git) [reply portals/Starbucks/portalscraper/chromedriver.exe:1]: No problem. I'm sure there's another way.

Comment (RocketGod-git) [reply portals/Starbucks/portalscraper/chromedriver.exe:1]: Updated the PR to include Linux.

Comment (amouly) [reply portals/Starbucks/portalscraper/chromedriver.exe:1]: @RocketGod-git still doesn't make sense to me.

I don't use Windows or Linux, why I would need to download those executables?

This should be an external dependency, installed from a dependency manager or something smart that allows me to install only the dependencies for my OS.

This project it's about an application to host an "evil portal", so you don't need to version also the tool to create the portals, that should be independent.

Comment (RocketGod-git) [reply portals/Starbucks/portalscraper/chromedriver.exe:1]: I'll just remove it for now until finished...

**Comment by RocketGod-git** [comment]
lol. Never tried sneaking company website logins before. So ya, I never saw this extension. I'll try that. Thanks.

**Comment by RogueMaster** [comment]
I think there is a file limit on the portal size, not sure exactly what it is, but, this file is much larger than the other portals.

**Comment by RocketGod-git** [comment]
Ya... I'll remove as much js as I can 🧐

**Comment by RogueMaster** [comment]
Limit is 20kb

**Comment by RocketGod-git** [comment]
I thought it was js carried over, but it's the images embedded with base64...

**Comment by RogueMaster** [comment]
Down to 48.2KB, still too big 😢 Maybe something like [tinyimg](https://tinypng.com/) can help...

**Comment by RocketGod-git** [comment]
Seems impossible to halve this 😒 lol. I'll try some more in a bit.

**Comment by RocketGod-git** [comment]
Got it to 19KB but couldn't keep the logo. Still looks good with the rest of the elements I could keep. Made a tool to help squash the html. 
[https://github.com/RocketGod-git/evilportal-htmlsquash](https://github.com/RocketGod-git/evilportal-htmlsquash)

**Pull request closed without merging**